examples/README.md

```diff
@@ -105,13 +105,30 @@ The [run](./run/) directory contains an example
 The [run](./run/) directory contains an example
 [TaskRun](../docs/Concepts.md#taskrun) with an embedded taskSpec, that pull a
 private image from `gcr.io`, see
-[`run/private-taskrun.yaml`](./run/private-taskrun.yaml).
-
-This _run_ requires the secrets from [`0-secrets.yaml`](`0-secrets.yaml`) and
-service accounts from [`1-bots.yaml`](`1-bots.yaml`) to be able to pull the
-private image.
+[`run/private-taskrun.yaml`](./run/private-taskrun.yaml). This _run_ requires the secrets and service accounts to be able to pull the
+private image. Required secret and service account for this run is configured in [`run/private-taskrun.yaml`](./run/private-taskrun.yaml) file itself.
 
 It uses `kubernetes.io/dockercfg` secret type but,
 `kubernetes.io/dockerconfigjson` and
 [Knative flavored credentials](https://github.com/knative/docs/blob/master/build/auth.md#guiding-credential-selection)
 are supported too.
+
+## Results
```

Comment:
ooo nice! maybe put a link to this section at the top?

Reply:
just bumping this :sweat_smile:

Reply:
README is pretty small so is there any value in having link at top?